boilerplate/app/screens/DemoShowroomScreen/DemoShowroomScreen.tsx

```diff
@@ -213,6 +216,18 @@ const $drawer: ViewStyle = {
   flex: 1,
 }
 
+const $drawerInner: ViewStyle = {
+  width: 350,
```

Comment:
[q] why is this a hard set value?

Reply:
`react-native-drawer-layout` somehow renders the contents leaning towards way right side when it's RTL. So `translatedX: -400` is applied to it, but the padding is off, padding didn't work so hardcoded the width to match the current contents.

Reply:
would getting the screen width be more exact or it should stay a hardcoded value?

Reply:
Hardcoded value. Tried pulling ratio from the screen, but considering the fixed content width would be better in my opinion. Ultimately we'd want `react-native-drawer-layout` to be fixed.

Reply:
ooof ok, do you mind opening an issue on the package and linking to it in the repo and this comment?

Reply:
Just found something else while verifying the issue. It might not be from the `react-native-drawer-layout`. Investigating.